UI/js-src/lsmb/DateTextBox.js

```diff
@@ -85,21 +85,33 @@ define([
         startup: function () {
             this.inherited(arguments);
 
+            /* Live insertion of date separators based on the lsmbConfig.Dateformat.
+             * The linter rule is disabled to allow assignment within the while() */
             /* eslint no-cond-assign: 0 */
-            /* Live insertion of date separators based on user selected format */
-            on(this.domNode, "keyup", lang.hitch(this, function (e) {
-                var value = domAttr.get(e.target, "value");
+            on(this.domNode, "keypress", lang.hitch(this, function (e) {
```

Comment:
Those errors should have been fatal. Can you please address?

Reply:
@ylavoie I'm wondering if that's something we can/should be asking from our contributors. Shouldn't we be happy with the contribution we're getting at it is?

Reply:
I provided him the fix